configapi/api_slice_mgmt.go

```diff
@@ -184,8 +186,8 @@ func NetworkSlicePostHandler(c *gin.Context, msgOp int) bool {
 		err = c.ShouldBindJSON(&request)
 	}
 	if err != nil {
-		logger.ConfigLog.Infof("err %v", err)
-		return false
+		logger.ConfigLog.Errorf("err %v", err)
```

Comment:
```suggestion
		logger.ConfigLog.Error(err.Error())
```

I would try to standardize the form that we use in this function. This seems sufficient.